project_project_category/models/project_project_category.py

```diff
@@ -0,0 +1,28 @@
+# © 2016-2019 Elico Corp (https://www.elico-corp.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import fields, models
+
+
+class ProjectProjectCategory(models.Model):
+    _name = 'project.project.category'
+
+    name = fields.Char('Name', required=True, translate=True, select=True)
```

Comment:
```suggestion
    name = fields.Char('Name', required=True, translate=True, index=True)
```
`select` is deprecated.